src/Components/QuizListContent.tsx

```diff
@@ -2,12 +2,29 @@ import styled from '@emotion/styled'
 import { css } from '@emotion/react'
 import { typography } from '../styles/theme'
 
-export const QuizListContent = (props: { type: 'Content' | 'End' }) => {
+interface propsType{
+  index:number;
+  title: string;
+  options: number;
+  date: string;
```

Comment:
for dates, it is better to pass data as Date type

Reply:
> for dates, it is better to pass data as Date type

Ok :)